daml-lf/scenario-interpreter/BUILD.bazel

```diff
@@ -54,6 +54,8 @@ daml_compile(
     visibility = ["//visibility:public"],
 )
 
+# TODO (MK) Figure out what to do about the benchmark.
```

Comment:
This is the main thing that I still need to figure out. I think this should really just go via the scenario runner instead of reimplementing things but that needs some mechanism for caching to make it work.